spec/gar_spec.lua

```diff
@@ -1554,12 +1553,62 @@ describe("gar", function()
 				}
 
 				local status, result =
-					pcall(gar.instantGatewayWithdrawal, from, gatewayAddress, vaultId, currentTimestamp)
+					pcall(gar.instantGatewayWithdrawal, from, gatewayAddress, protectedVaultId, currentTimestamp)
 
 				assert.is_false(status)
-				assert.matches("This gateway is leaving and this vault cannot be instantly withdrawn.", result)
+				assert.matches("Gateway operator vault cannot be instantly withdrawn.", result)
 			end)
 
+			it(
+				"should allow gateway operator to instantly withdraw their vault when leaving, if it is not the protected operator vault",
```

Comment:
Do we have the analogous test case to ensure we're preventing withdrawal of the minimum stake vault?

Reply:
yep - the one above this one https://github.com/ar-io/ar-io-network-process/pull/422/files#diff-f6579ae3c0194494060e6eeb361f3cdcf1041f242e52346974bd890b5450a614R1529-R1560